udf-catalogue.tex

```diff
@@ -412,11 +417,33 @@ \subsubsection{ivo\_epoch\_prop(ra, dec, parallax, pmra, pmdec,
 \end{lstlisting}
 \becomes [7.6040614046279735, 11.793270382827929, 125.01993165584682,\\
 300.09877325973605, -428.934593565712, 52.50880381775256]
+
 \example \begin{lstlisting}
 ivo_epoch_prop(7.606083572, 11.79044105, 125,
   NULL, NULL, NULL, 2016.0, 1992.25)
 \end{lstlisting}
-\becomes [7.606083572000001, 11.79044105, 125.0, 0.0, 0.0, 0.0]
+\becomes [7.606083572, 11.79044105, 125, NULL, NULL, NULL]
+
+\example \begin{lstlisting}
+ivo_epoch_prop(7.606083572, 11.79044105, NULL,
+  300, -428.8, 52.51, 2016.0, 1992.25)
+\end{lstlisting}
+\becomes [7.604061727024453, 11.79326993174991, NULL, \\
+  300.0030911918108, -428.79783514643105, 52.51]
+
+\example \begin{lstlisting}
+ivo_epoch_prop(7.606083572, 11.79044105, 125,
+  21, NULL, NULL, 2016.0, 1992.25)
+\end{lstlisting}
+\becomes [7.606083572, 11.79044105, 125.0, NULL, NULL, NULL]
```

Comment:
This example is a bit weird - `pmra` defined but `pmdec` NULL on input.  It's not a very likely case, and the most sensible output is debatable - you could argue that instead of nulling out both PM elements on output it should leave the known `pmra` intact and return exactly the same output as input.  Was it actually intended like this?  I wonder if it's supposed to be
```
ivo_epoch_prop(7.606083572, 11.79044105, 125,  NULL, NULL, 21, 2016.0, 1992.25)
```
instead - which I think would be a more instructive example.